src/state/user/reducer.ts

```diff
@@ -55,6 +57,7 @@ function pairKey(token0Address: string, token1Address: string) {
 }
 
 export const initialState: UserState = {
+  buyFiatFlowCompleted: false,
```

Comment:
since we don't have redux migrations yet, this initial state will technically just be `undefined`

Reply:
yeah im saying you should just put `undefined` here to be more accurate

Reply:
ooh sg can do, can change back to false after the migration left a todo.